zerver/migrations/0135_realm_uploads_quota_gb.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.6 on 2017-12-31 19:01
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0134_scheduledmessage'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='realm',
+            name='uploads_quota_gb',
```

Comment:
For consistency, let's make it "upload", singular, everywhere (or everywhere that's changing in these commits, anyway). Consistency avoids having to guess, or remember, where we used one and where the other.